query-engine/connector-test-kit-rs/query-tests-setup/src/lib.rs

```diff
@@ -171,7 +170,7 @@ fn run_relation_link_test_impl(
                     .await
                     .unwrap();
 
-                test_fn(&runner, &dm_with_params_json).await.unwrap();
+                test_fn(&runner, &dm).await.unwrap();
```

Comment:
It doesn't have to be done in _this_ PR but it'd be great if we could make it easier to debug failing tests. I see two ways this could be done:

- Either pass `i` (the datamodel index) to the test runner so that we can programmatically pick a single test
- Or add an env var that'd do it for us (eg: `RELATION_TEST_IDX=12`)

For that to be useful, we could have a meaningful error that tells us which test index failed exactly. Workflow would be similar to `RUST_BACKTRACE`:

1. Run test and read which index failed
2. Re-run test with `RELATION_TEST_IDX=<index_given_in_error_message>`

Wdyt?

Reply:
yes, true! One alternative I thought of would be having a tracing span with the index, so all the log messages in the failed test output have the index, but your solution goes further (and is complementary). Let me check if I can make it happen fast.

Reply:
I got started on that, coming in a separate PR today or tomorrow.

Reply:
https://github.com/prisma/prisma-engines/pull/3895